frontend/src/ui/Blocks/Series.tsx

```diff
@@ -125,25 +125,49 @@ const ReadySeriesBlock: React.FC<ReadyProps> = ({
     const finalTitle = title ?? (showTitle ? series.title : undefined);
 
     const events = series.events.filter(event =>
-        !isPastLiveEvent(event.syncedData?.endTime ?? null, event.isLive));
+        !isPastLiveEvent(event.syncedData?.endTime ?? null, event.isLive)
+        && !isUpcomingEvent(event.syncedData?.startTime ?? null));
+
+    const upcomingEvents = series.events.filter(event =>
+        isUpcomingEvent(event.syncedData?.startTime ?? null));
 
     const sortedEvents = [...events];
-    sortedEvents.sort(match(order, {
-        "NEW_TO_OLD": () => compareNewToOld,
-        "OLD_TO_NEW": () => compareOldToNew,
-    }, unreachable));
+    sortedEvents
+        .sort(match(order, {
+            "NEW_TO_OLD": () => compareNewToOld,
+            "OLD_TO_NEW": () => compareOldToNew,
+        }, unreachable))
+        .sort((a, b) => +b.isLive - +a.isLive);
+
+    if (upcomingEvents.length === 1) {
+        sortedEvents.unshift(upcomingEvents[0]);
+    } else {
+        upcomingEvents.sort(compareOldToNew);
+    }
 
     const eventsUI = events.length === 0
         ? t("series.no-events")
-        : <VideoGrid>
-            {sortedEvents.map(
-                event => <GridTile
-                    key={event.id}
-                    active={event.id === activeEventId}
-                    {...{ basePath, event }}
-                />,
-            )}
-        </VideoGrid>;
+        : <>
+            {upcomingEvents.length > 1
+                && <UpcomingEventsGrid>
+                    {upcomingEvents.map(
+                        event => <GridTile
+                            key={event.id}
+                            active={event.id === activeEventId}
+                            {...{ basePath, event }}
+                        />,
```

Comment:
This part seems duplicate to the one below. Maybe add a helper above like

```javascript
const eventsToTiles = (events: Event[]) => events.map(event => <GridTile .../>);
```

And then use `eventsToTiles(upcomingEvents)` and `eventsToTiles(sortedEvents)` below.